t/01-detect.t

```diff
@@ -42,12 +42,16 @@ foreach my $ua ( sort ( keys %{$tests} ) ) {
             'device_beta', 'device_name',  'device_string',  'engine',
             'engine_beta', 'engine_string', 'language', 'os', 'os_beta',
             'os_string', 'robot', 'robot_beta', 'robot_name', 'robot_string',
+            'webview',
             ) {
-            if ( $test->{$method} ) {
-                cmp_ok(
-                    $detected->$method || q{}, 'eq', $test->{$method},
-                    "$method: $test->{$method}"
-                );
+            if ( exists $test->{$method} ) {
+                if ( defined $test->{$method} ) {
+                    eq_or_diff( $detected->$method, $test->{$method}, "$method: $test->{$method}");
+                }
+                else {
+                    eq_or_diff( $detected->$method, $test->{$method}, "$method: undef" );
+                    #ok( ! defined $detected->$method, "$method should return undef" );
```

Comment:
Should we remove this commented line? Looks like leftover debugging.

Reply:
Ohh, yes, please remove comment line
I forgot, sorry

Reply:
I deleted this line and pushed to my repo
You can see this